### PR TITLE
yubikey-manager: 5.0.0 -> 5.0.1

### PR DESCRIPTION
**Pull request by austinbutler**
###### Description of changes

https://github.com/Yubico/yubikey-manager/releases/tag/5.0.1
https://github.com/Yubico/yubioath-flutter/releases/tag/6.1.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by austinbutler** [comment]
Result of `nixpkgs-review pr 215271` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>gen-oath-safe</li>
    <li>yubikey-manager</li>
    <li>yubioath-flutter</li>
  </ul>
</details>

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 215271` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>gen-oath-safe</li>
    <li>yubikey-manager</li>
  </ul>
</details>

**Comment by wegank** [comment]
@ofborg build yubikey-manager yubikey-manager.passthru.tests yubioath-flutter yubioath-flutter.passthru.tests

**Comment by Lassulus** [comment]
looks fine, just running a quick nix-review

**Comment by wegank** [comment]
```
error: hash mismatch in fixed-output derivation '/nix/store/1j85hc0sv185p7gq5l9gqv6g946bi15r-yubioath-flutter-6.1.0-deps-flutter-v3.3.8-aarch64-linux.tar.gz.drv':
         specified: sha256-Upe0cEDG02RJD50Ht9VNMwkelsJHX8zOuJZssAhMuMY=
            got:    sha256-lKER4+gcyFqnCvgBl/qdVBCbUpocWUnXGLXsX82MSy4=
error: 1 dependencies of derivation '/nix/store/b1s4vxzwxq0jvimxyv035y8y2b1pcq93-yubioath-flutter-6.1.0.drv' failed to build
```

File: pkgs/applications/misc/yubioath-flutter/default.nix [hunk 1]
```diff
@@ -10,19 +10,19 @@
 }:
 let
   vendorHashes = {
-    x86_64-linux = "sha256-BwhWA8N0S55XkljDKPNkDhsj0QSpmJJ5MwEnrPjymS8=";
-    aarch64-linux = "sha256-T1aGz3+2Sls+rkUVDUo39Ky2igg+dxGSUaf3qpV7ovQ=";
+    x86_64-linux = "sha256-Upe0cEDG02RJD50Ht9VNMwkelsJHX8zOuJZssAhMuMY=";
+    aarch64-linux = "sha256-Upe0cEDG02RJD50Ht9VNMwkelsJHX8zOuJZssAhMuMY=";
```
Comment (wegank) [inline pkgs/applications/misc/yubioath-flutter/default.nix:14]:
```suggestion
    aarch64-linux = "sha256-lKER4+gcyFqnCvgBl/qdVBCbUpocWUnXGLXsX82MSy4=";
```

**Comment by Lassulus** [comment]
Result of `nixpkgs-review pr 215271` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>gen-oath-safe</li>
    <li>yubikey-manager</li>
    <li>yubioath-flutter</li>
  </ul>
</details>

**Review by wegank — APPROVED** [review]

**Review by pinpox — APPROVED** [review]
Builds and executes fine for me!

File: pkgs/applications/misc/yubioath-flutter/default.nix (hunk 1 above)
Comment (austinbutler) [reply pkgs/applications/misc/yubioath-flutter/default.nix]: Thanks! I thought using `pkgsCross` would work but apparently not. Is there a way to get the SHA for an arch I don't have a device for without ofborg?

**Comment by dotlambda** [comment]
Can we backport this to fix https://github.com/NixOS/nixpkgs/issues/214436 on 22.11?

**Pull request merged**